mithril-explorer/src/utils.js

```diff
@@ -78,6 +78,20 @@ function getCExplorerUrlForPool(network, partyId) {
   return url;
 }
 
+function calculatesProcessDuration(startTime) {
+  const duration = performance.now() - startTime;
+  const minutes = Math.floor(duration / 60000);
+  const seconds = Math.floor((duration % 60000) / 1000);
+  return(
+    minutes > 0 ? `${minutes} minutes and ${seconds} seconds` : `${seconds} seconds`
+  );
+}
+
+function retrieveAggregatorNetworkFromUrl(aggregatorUrl) {
```

Comment:
Maybe name it differently:
```suggestion
function computeAggregatorNetworkFromUrl(aggregatorUrl) {
```